ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -21,6 +21,7 @@ import {services} from '../../../shared/services';
 import {getResolvedTemplates} from '../../../shared/template-resolution';
 
 import './workflow-node-info.scss';
+import {TIMESTAMP_KEYS} from '../../../shared/use-timestamp';
```

Comment:
same as above

Reply:
This one was individually fixed separately from the rest in https://github.com/argoproj/argo-workflows/pull/13494